common-bsp/recipes-kernel/device-tree-overlays/device-tree-overlays_git.bb

```diff
@@ -0,0 +1,33 @@
+DESCRIPTION = "Device Tree Overlays for bb.org boards"
+
+LICENSE = "MIT"
```

Comment:
Problem for GPLv2 is the source for LIC_FILES_CHKSUM as upstream repository does not ship with any parseable license information. Suggestions?